src/Mod/Sketcher/Gui/DrawSketchHandlerFillet.h

```diff
@@ -333,7 +333,12 @@ class DrawSketchHandlerFillet: public DrawSketchHandler
     {
         Gui::Selection().rmvSelectionGate();
         Gui::Selection().addSelectionGate(new FilletSelection(sketchgui->getObject()));
-        return QString::fromLatin1("Sketcher_Pointer_Create_Fillet");
+        if (filletType == 0) {

```

Comment:
It is better to use the enum name for clarity : `filletType == SimpleFillet`
In the rest of the file it is not done like this though. Perhaps you can edit the 2 other places where there is filletType == x.

Reply:
done

Reply:
And this is one of the reasons why we should almost always prefer an ```enum class``` instead of the legacy ```enum```, if it were the former, that line of code would have given a compiler error and the author would not have submitted it in the first place.

A maintainer should consider whether to, in a separate PR, upgrade the enum.